partials/tabs/home.html

```diff
@@ -131,3 +131,9 @@ <h2 class="fine">get started</div>
         </div>
     </div>
 </section>
+<script>
+$(document).ready(function(){
+  $('ul.tabs').tabs();
+});
+
+</script>
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/home.html
+++ b/tmp/tmp8v872ato/partials/tabs/home.html
@@ -136,4 +136,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```